pkg/storage/registry/spaces/Readme.md

```diff
@@ -0,0 +1,52 @@
+# Spaces Registry
+
+The spaces registry recognizes individual spaces instead of storage providers.
+While it is configured with a list of storage providers, it will query them for all storage spaces and use the space ids to resolve id based lookups.
+Furthermore, path based lookups will take into account a path templvate to present a human readable file tree.
```

Comment:
> account a path template to present

Reply:
Done in https://github.com/cs3org/reva/pull/2383